cypress/features/regression/designSystem/drawer.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Drawer component
+  I want to test Drawer component
+
+  Background: Open Drawer component page
+    Given I open Design Systems controlled "Drawer" component docs page
+
+  @positive
+  Scenario: Colapsing and Expanding Drawer with Controls
+    When I click on "with-controls" Drawers arrow 1 time
+    Then Drawers "with-controls" sidebar should have class closed
+      And toggle "with-controls" Drawers icon switched orientation to closed
+      And Drawer "with-controls" sidebar text is not visible
+    When I click on "with-controls" Drawers arrow 1 time
```

Comment:
those tests should be separate

Reply:
I disagree, we need to test the scenario where two clicks are carried out on the toggle button/icon to see the drawer behaviour opening -> open and then closing -> closed.
It is not the same as if I were to render drawer opened and click once to close or opposite.

Reply:
fixed.